### PR TITLE
Export feature

### LINKED ISSUE #83: Export contacts
Allow users to export contacts into a json file on their local device.

### PR DESCRIPTION
**Pull request by myrilla12**
closes #83 

### Description
Exports addressbook.json to users local device under data/addressbook.json upon command `export`

### Key Changes
- Add ExportCommand
- Add tests
- Update user guide & help window command summary

### Potential Extensions
- Make UI more intuitive (e.g. pop-up window with more detailed notes on where user can find their addressbook.json file)

### CODE REVIEW

**Comment by myrilla12** [comment]
codecov coverage unsuccessful is due to removal of irrelevant test case

File: src/test/java/seedu/address/logic/commands/ExportCommandTest.java
[the remark this answers is not in the document]
Comment (myrilla12) [reply src/test/java/seedu/address/logic/commands/ExportCommandTest.java:1]: This was originally the case but you can refer to commit #2a935c2e for more details on why the test was removed.

File: src/main/java/seedu/address/logic/commands/ExportCommand.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    public static final String MESSAGE_EXPORT_ACKNOWLEDGEMENT = "Exporting addressbook.json under "
+            + "file path data/addressbook.json in whichever folder addressbook.jar is in";
+    private static final String ADDRESSBOOK_FILE_PATH = "data/addressbook.json";
+
+    @Override
+    public CommandResult execute(Model model) {
+        requireNonNull(model);
+
+        try {
+            File file = new File(ADDRESSBOOK_FILE_PATH);
```
Comment (OttoTan) [inline src/main/java/seedu/address/logic/commands/ExportCommand.java:26]: I might be missing it but how does the JSON file get placed into the folder the addressbook,jar file is in?

File: src/main/java/seedu/address/logic/commands/ExportCommand.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import seedu.address.model.Model;
+
+/**
+ * Exports addressbook.json into user's local device.
+ */
+public class ExportCommand extends Command {
+
+    public static final String COMMAND_WORD = "export";
+
+    public static final String MESSAGE_EXPORT_ACKNOWLEDGEMENT = "Exporting addressbook.json under "
```
[the remark this answers is not in the document]
Comment (myrilla12) [reply src/main/java/seedu/address/logic/commands/ExportCommand.java]: I've updated the message to mention MediContact!

File: src/main/java/seedu/address/logic/commands/ExportCommand.java (hunk 1 above)
Comment (myrilla12) [reply src/main/java/seedu/address/logic/commands/ExportCommand.java:26]: The path data/addressbook.json is relative to the location where addressbook.jar is run.

**Review by OttoTan — APPROVED** [review]
LGTM

File: src/test/java/seedu/address/logic/commands/ExportCommandTest.java
[the remark this answers is not in the document]
Comment (ritvi12) [reply src/test/java/seedu/address/logic/commands/ExportCommandTest.java:1]: ok! Makes sense

**Review by ritvi12 — APPROVED** [review]
LGTM!

**Pull request merged**